Formula/meraculous.rb

```diff
@@ -0,0 +1,63 @@
+class Meraculous < Formula
+  desc "Whole genome assembler for NGS reads geared for large genomes"
+  homepage "https://jgi.doe.gov/data-and-tools/meraculous/"
+  # doi "10.1371/journal.pone.0023501"
+  # tag "bioinformatics
```

Comment:
Please delete these two lines and add this line immediately after the `class` line before `desc`.
```
# cite Chapman_2011: "https://doi.org/10.1371/journal.pone.0023501"
```